easybuild/easyconfigs/o/Octave/Octave-4.2.1-intel-2017a-mt.eb

```diff
@@ -6,14 +6,15 @@ homepage = 'http://www.gnu.org/software/octave/'
 description = """GNU Octave is a high-level interpreted language, primarily intended for numerical computations."""
 
 toolchain = {'name': 'intel', 'version': '2017a'}
-toolchainopts = {'cstd': 'c++11'}
+toolchainopts = {'cstd': 'c++11', 'pic': True}
```

Comment:
Can you clarify the need to enable `-fPIC`?

Reply:
I believe this occurs because the `image` extension package has a `.so` component.  Without this flag, I got an `ld` error.  In other words, I think `pic` is required here for the same reason that (say) `Python` requires it.